#ifndef ABB_SERVICE_MESSAGES
#define ABB_SERVICE_MESSAGES



#include "abb_node/robot_GetCartesian.h"
#include "abb_node/robot_SetSpeed.h"
#include "abb_node/robot_GetCartesianRequest.h"
#include "abb_node/robot_SetSpeedRequest.h"
#include "abb_node/robot_GetCartesianResponse.h"
#include "abb_node/robot_SetSpeedResponse.h"
#include "abb_node/robot_GetJoints.h"
#include "abb_node/robot_SetTool.h"
#include "abb_node/robot_GetJointsRequest.h"
#include "abb_node/robot_SetToolRequest.h"
#include "abb_node/robot_GetJointsResponse.h"
#include "abb_node/robot_SetToolResponse.h"
#include "abb_node/robot_IsMoving.h"
#include "abb_node/robot_SetTrackDist.h"
#include "abb_node/robot_IsMovingRequest.h"
#include "abb_node/robot_SetTrackDistRequest.h"
#include "abb_node/robot_IsMovingResponse.h"
#include "abb_node/robot_SetTrackDistResponse.h"
#include "abb_node/robot_Ping.h"
#include "abb_node/robot_SetVacuum.h"
#include "abb_node/robot_PingRequest.h"
#include "abb_node/robot_SetVacuumRequest.h"
#include "abb_node/robot_PingResponse.h"
#include "abb_node/robot_SetVacuumResponse.h"
#include "abb_node/robot_SetCartesian.h"
#include "abb_node/robot_SetWorkObject.h"
#include "abb_node/robot_SetCartesianRequest.h"
#include "abb_node/robot_SetWorkObjectRequest.h"
#include "abb_node/robot_SetCartesianResponse.h"
#include "abb_node/robot_SetWorkObjectResponse.h"
#include "abb_node/robot_SetComm.h"
#include "abb_node/robot_SetZone.h"
#include "abb_node/robot_SetCommRequest.h"
#include "abb_node/robot_SetZoneRequest.h"
#include "abb_node/robot_SetCommResponse.h"
#include "abb_node/robot_SetZoneResponse.h"
#include "abb_node/robot_SetDIO.h"
#include "abb_node/robot_SpecialCommand.h"
#include "abb_node/robot_SetDIORequest.h"
#include "abb_node/robot_SpecialCommandRequest.h"
#include "abb_node/robot_SetDIOResponse.h"
#include "abb_node/robot_SpecialCommandResponse.h"
#include "abb_node/robot_SetJoints.h"
#include "abb_node/robot_Stop.h"
#include "abb_node/robot_SetJointsRequest.h"
#include "abb_node/robot_StopRequest.h"
#include "abb_node/robot_SetJointsResponse.h"
#include "abb_node/robot_StopResponse.h"


#endif
